packages/gluestack-ui/src/r5/data-display/fhir-value.tsx

```diff
@@ -0,0 +1,22 @@
+import { FhirValueRendererProps } from "@bonfhir/react/r5";
+import { Text } from "@gluestack-ui/themed";
+import { ReactElement } from "react";
+
+export function GlueStackFhirValue(
+  props: FhirValueRendererProps<GlueStackFhirValueProps>,
+): ReactElement | null {
+  return (
+    <Text
+      className={props.className}
```

Comment:
Does gluestack actually support className? Only when running in web, maybe?

Reply:
very good point. There was no error when testing in the ecpo app, but I will remove it